e2e/index.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+var TRACE_API_KEY_TEST = 'api-key'
+var TRACE_SERVICE_NAME_TEST = 'service-name'
+
+var childProcessTest = require('./utils/childProcessTest')
+var test = require('tape')
```

Comment:
@peteyy or do you mean, why do we need it here, when we have no assertions? thats a good point